### ISSUE TITLE
Add tests for t.w.http.stringToDatetime

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8844]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8844|
|Type|enhancement|
|Created|2016-09-29 17:05:50Z|

https://github.com/twisted/twisted/pull/552

<details><summary>Searchable metadata</summary>

```
trac-id__8844 8844
type__enhancement enhancement
reporter__scop scop
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__None None
time__1475168750641484 1475168750641484
changetime__1475187433412649 1475187433412649
version__None None
owner__Mark_Williams__markrwilliams_____ Mark Williams <markrwilliams@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_2"></a>|@markrwilliams commented|
|-|-|

A simple and useful new test.

I'll merge when the builds are green.  Thanks!

**Comment by twisted-trac** [comment]
|<img alt="Ville Skyttä's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|Ville Skyttä commented|
|-|-|

In changeset ae40b8c1c39f622001c00517a04d5f0cf7bbea65
```
#!CommitTicketReference repository="" revision="ae40b8c1c39f622001c00517a04d5f0cf7bbea65"
Add topfile for #8844
```

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_4"></a>|@markrwilliams set owner to @markrwilliams<br>@markrwilliams set status to `closed`|
|-|-|

In changeset 2a9091c1fdaceaf3ffbc32edcdb7cb6b953a1bef
```
#!CommitTicketReference repository="" revision="2a9091c1fdaceaf3ffbc32edcdb7cb6b953a1bef"
Merge pull request #552 from scop/8844-scop-http-date-tests

Merge scop/8844-scop-http-date-tests: Add t.w.http.stringToDatetime tests

Author: scop
Reviewer: markrwilliams
Fixes: #8844
```
